packages/replay-next/pages/variables.css

```diff
@@ -48,7 +48,7 @@
   --primary-accent: #01acfd;
   --primary-accent-hover: #0194ff;
 
-  --secondary-accent: var(--secondary-accent);
```

Comment:
How was this supposed to have worked before 😕

Reply:
Yeah, I fat fingered this in a previous PR and did an emergency one-liner PR when I saw it.